docker-compose.yaml

```
@@ -65,6 +66,8 @@ x-worker: &base_worker_setup
 services:
   db:
     image: postgis/postgis:17-3.5
+    ports:
+      - 127.0.0.1:5432:5432
```

Is this okay @thenav56 ?

---

This is fine for development, but I would also recommend adding environment variable to allow custom ports
```suggestion
    ports:
      - 127.0.0.1:${MAPSWIPE_DB_EXPOSE_PORT:-5432}:5432
```